src/runtime/cpu_device_api.cc

```diff
@@ -100,6 +100,21 @@ void CPUDeviceAPI::FreeWorkspace(Device dev, void* data) {
   dmlc::ThreadLocalStore<CPUWorkspacePool>::Get()->FreeWorkspace(dev, data);
 }
 
+TVM_REGISTER_GLOBAL("device_api.cpu.mem_copy").set_body([](TVMArgs args, TVMRetValue* rv) {
```

Comment:
I think it is better to remove this - it was just for demonstration purpose.

Reply:
Yeah note that we do have similar functionality in the TVM C API (`TVMArrayCopyFromTo`), so probably it might be less relevant for our specific usecase?

Reply:
Done.